platform/representation/src/MCTRepresentation.js

```diff
@@ -159,10 +162,11 @@ define(
                         path = representation && getPath(representation),
                         uses = ((representation || {}).uses || []),
                         canRepresent = !!(path && domainObject),
+                        canEdit = !!(domainObject && domainObject.hasCapability('editor')),
                         id = domainObject && domainObject.getId(),
                         key = $scope.key;
 
-                    if (unchanged(canRepresent, id, key)) {
+                    if (unchanged(canRepresent, canEdit, id, key)) {
```

Comment:
...and we do! #302. Once that is resolve I imagine these changes will no longer be necessary.